lib/utils.js

```diff
@@ -92,7 +92,7 @@ exports.findAvailablePort = function (app, callback) {
           callback(port)
         } else {
           // User answers no - exit
-          console.log('\nYou can set a new default port in server.js, or by running the server with PORT=XXXX')
+          console.log('\nYou can set a new default port in app/config.json, or by running the server with PORT=XXXX')
```

Comment:
```suggestion
```

Reply:
let's drop the whole line - telling people to set a port is not necessarily the solution to the problem they're having. In future we could have docs about ports/running multiple prototypes, and link to that